### PR TITLE
Pull reqest test.

### PR DESCRIPTION
**Pull request by vlad1m1r-k**
Main.java fixed.

### CODE REVIEW

File: src/session1/Main.java [hunk 1]
```diff
@@ -5,6 +5,6 @@
  */
 public class Main {
     public static void main(String[] args) {
-        System.out.println("qqq");
+        System.out.println("Hello world!!!");
```
Comment (alex-zr) [inline src/session1/Main.java]: Please change your message to Hello World

Comment (alex-zr) [inline src/session1/Main.java]: Please change your message to Hello World

**Comment by alex-zr** [comment]
comment

File: src/session1/Main.java (hunk 1 above)
Comment (vlad1m1r-k) [reply src/session1/Main.java]: Done.

**Pull request merged**